XcodeServerSDK/Utils/Logging.swift

```diff
@@ -50,7 +50,7 @@ public class ConsoleLogger: Logger {
     }
     
     public func log(message: String) {
-        println(message)
+        print(message)
```

Comment:
Turns out `true` for `appendNewline` is actually the default. So all good in the hood.